crates/turbopack-dev-server/src/source/asset_graph.rs

```diff
@@ -148,28 +150,69 @@ async fn expand(
     Ok(map)
 }
 
+#[turbo_tasks::value(serialization = "auto_for_input")]
+#[derive(Hash, PartialOrd, Ord, Debug, Clone)]
+struct UnresolvedAsset(AssetVc);
```

Comment:
```suggestion
#[derive(Hash, PartialOrd, Ord, Debug, Clone, TaskInput)]
struct UnresolvedAsset(AssetVc);
```

Then you don't need to wrap it in a `Value`.

Reply:
Sadly that doesn't work, as TaskInput would resolve the contained RawVc, which we don't want.

I would love to have the ability to add `#[turbo_tasks::unresolved]` to arguments (and properties within TaskInput).

Reply:
Can you add a comment that explains this?